back/src/adapters/primary/routers/apiKeyAuthRouter/contactEstablishmentPublicV2.e2e.test.ts

```diff
@@ -118,16 +118,16 @@ describe("POST contact-establishment public V2 route", () => {
       headers: {
         authorization: authToken,
       },
-      body: {} as any,
+      body: { ...contactEstablishment, siret: "wrong" } as any,
```

Comment:
possible d'enlever `any`:
```suggestion
      body: { ...contactEstablishment, siret: "wrong" },
```